googletest/include/gtest/internal/gtest-internal.h

```diff
@@ -1282,6 +1282,21 @@ class FlatTuple
   }
 };
 
+class AdditionalMessage {
```

Comment:
Lets make this a struct with value being public, there is really no reason for access control.

Reply:
I merged the functionality in AdditionalMessage into ConstCharPtr and used the same style for the new code. It could have been done the other way too I guess, not sure what makes most sense - but this way results in a smaller diff.